src/main/java/sirius/db/mongo/constraints/MongoFilterFactory.java

```diff
@@ -278,4 +278,18 @@ public MongoConstraint text(String value) {
 
         return new MongoConstraint("$text", new Document("$search", value));
     }
+
+    /**
+     * Builds a constraint regarding the {@code $size} of a (list) mapping.
+     * <p>
+     * See <a href="https://docs.mongodb.com/manual/reference/operator/query/size/">the manual</a> for usage information
+     * and restrictions concerning the operator.
+     *
+     * @param mapping the mapping to filter
+     * @param value the target size to compare against
+     * @return the generated constraint
+     */
+    public MongoConstraint size(Mapping mapping, Integer value) {
```

Comment:
shouldn't we call this s.th. like "hasListSize" or the like?

And why is value nullable? what is the semantics of null here? Should be either an int or explained